### DISCUSSION TITLE
How to jump back from go to definition

### DISCUSSION THREAD

**Discussion by jorgsk** [LunarVim/LunarVim#2706] [General]
I'm used to the workflow of using ctrl-] for goto definition, and ctrl-t for jumping back from the tag stack.

In LunarVim, the default is to use "gd" to go to a definition (although ctrl-] also works for some reason). However, ctrl-t opens a terminal and I can't find any key binding for jumping back in the call stack. ctrl-o is not a working substitute since it doesn't follow the tag stack.

My question is then, with LunarVim, how can I use ctrl-T to jump back in the tag stack?

**Comment by jorgsk** [comment]
So, I found a workaround for the moment. In my config.lua:

```
lvim.keys.normal_mode["<C-t>"] = ":pop<cr>"
```

I'd prefer if this was the default behavior for C-t, given that this is how it works in Neovim and in Vim for many years, and that invoking a terminal could get some other key-press combination.

**Reply by zalegrala** [reply]
Using `ctrl+o` works for me.

**Reply by jorgsk** [reply]
Ctrl+o jumps back to a previous position, but at least on my install it does not jump in the tag stack, which is what I need when navigating up and down a call tree while taking excursions here and there.
